i18n/shared/src/main/scala/io/udash/i18n/types.scala

```diff
@@ -1,8 +1,20 @@
 package io.udash.i18n
 
-import com.avsystem.commons.serialization.transparent
+import com.avsystem.commons.serialization.{GenCodec, transparent}
 
 @transparent case class Lang(lang: String) extends AnyVal
+object Lang {
+  implicit val codec: GenCodec[Lang] = GenCodec.materialize[Lang]
+}
```

Comment:
Well, `HasGenCodec` has a lot of limitations, so I don't want to encourage it so much...